source/clear-linux/get-started/virtual-machine-install/virtualbox-cl-installer.rst

```diff
@@ -3,256 +3,311 @@
 
+#. Delete the originally downloaded compressed archive file.
 
 Create a new |VB| virtual machine
 *********************************
 
-A new :abbr:`VM (Virtual Machine)` needs to be created in |VBM| for |CL| to be installed onto. 
-
-General instructions for creating a virtual machine and details about using 
-different settings are available on the 
+A new :abbr:`VM (Virtual Machine)` needs to be created in |VBM| where |CL|
+will be installed. General instructions for creating a virtual machine and
+details about using different settings are available on the
 `VirtualBox manual section on Creating a VM`_.
 
-
 #. Launch the |VBM| from your host system.
 
+#. Click the *New* button to create a new VM.
 
-#. Click the *New* button to create a new VM. 
-
-   .. image:: ./figures/vbox/vbox-new-vm.png
+   .. figure:: figures/vbox/virtualbox-cl-installer-02.png
+      :scale: 100%
       :alt: Create a new VM in VirtualBox
 
+      Figure 2: Create a new VM in VirtualBox
 
-#. A *Create Virtual Machine* window will appear. 
+#. A *Create Virtual Machine* window will appear.
    Select the following settings:
-   
+
+   - Name: Enter a name (e.g. ClearLinuxOS).
    - Type: **Linux**
    - Version: **Linux 2.6 / 3.x / 4.x (64-bit)**
-   - Memory size: **1024 MB** (this can be adjusted appropriately)
-   - Hard disk: **Create a virtual hard disk now**
 
-   .. image:: ./figures/vbox/vbox-create-vm-new-disk.png
-      :alt: Create a new VM in VirtualBox with a new disk
+#. Enter a memory size.
 
+   - Memory size default: **1024 MB** (Adjust appropriately.)
 
-#. Click the *Create* button.
+#. In :guilabel:`Hard disk`, select
+   :guilabel:`Create a virtual hard disk now`.
 
-#. A *Create Virtual Hard Disk* window will appear. 
-   Select the following settings:
+   .. figure:: figures/vbox/virtualbox-cl-installer-03.png
+      :scale: 100%
+      :alt: Create a virtual hard disk
+
+      Figure 3: Create a virtual hard disk
+
+#. Select :guilabel:`Create`.
+
+#. In :guilabel:`Hard disk file type`, select
+   :guilabel:`VDI (VirtualBox Disk Image)`.
+
+   .. figure:: figures/vbox/virtualbox-cl-installer-04.png
+      :scale: 100%
+      :alt: Hard disk file type
+
+      Figure 4: Hard disk file type
+
+#. Select :guilabel:`Next`
+
+#. In :guilabel:`Storage on physical hard disk`, select
+   :guilabel:`Dynamically allocated`.
+
+   .. figure:: figures/vbox/virtualbox-cl-installer-05.png
+      :scale: 100%
+      :alt: Storage on a physical hard disk
 
-   - File size: **8.00 GB** (this can be adjusted appropriately)
-   
-   - Hard disk file type: **VDI (Virtual Box Disk Image)**
+      Figure 5: Storage on a physical hard disk
 
-   - Storage on physical hard disk: **Dynamically allocated** 
+#. Select :guilabel:`Next`.
 
-   .. image:: ./figures/vbox/vbox-create-disk.png
-      :alt: Create a new virtual hard disk in VirtualBox
+#. In :guilabel:`File location and size`, select a File size: **8.00 GB**
```

Comment:
8.00 GB is probably ok for a non-GUI installation, but if they want GUI, they'll be starting out with less than 3 GB free. It's probably easier to allocate a bigger disk up front, especially since VDI only takes up the space that's actually in use.

Reply:
OK, so do you think a note is in order like:
"If installing the `live-desktop` (GUI), ... you can start with 3GB"?

Reply:
I'd probably just bump the starting point up to 30 GB or so, but leave your note about how this can be customized. There's little harm in over-sizing the virtual hard drive at creation, but it's a pain to embiggen it later. That way the person who breezes through this guide in a hurry is more likely to remain happy with the results.

Reply:
See latest commit: https://github.com/clearlinux/clear-linux-documentation/blob/7d43f265d9ae5341b41ef28155e1b7e608835417/source/clear-linux/get-started/virtual-machine-install/virtualbox-cl-installer.rst

Reply:
Ah, ok, that should work then.

Reply:
@bwarden Thanks for the review.